src/components/scenes/CreateWalletCompletionScene.tsx

```diff
@@ -184,7 +218,10 @@ const CreateWalletCompletionComponent = (props: Props) => {
             scrollIndicatorInsets={SCROLL_INDICATOR_INSET_FIX}
           />
 
-          <SceneButtons primary={{ label: lstrings.string_next_capitalized, disabled: !done, onPress: handleNext }} />
+          <SceneButtons
+            primary={{ label: lstrings.string_next_capitalized, disabled: !done, onPress: handleNext }}
+            secondary={{ label: lstrings.migrate_label, disabled: !done, onPress: handleMigrate }}
```

Comment:
I missed this on the first round of review. This should only be visible for the import flow so it should be dependent on the importText being present